app/controllers/concerns/remember_device_concern.rb

```diff
@@ -73,7 +73,7 @@ def handle_valid_remember_device_cookie(remember_device_cookie:)
     user_session[:auth_method] = TwoFactorAuthenticatable::AuthMethod::REMEMBER_DEVICE
     mark_user_session_authenticated(:device_remembered)
     handle_valid_remember_device_analytics(cookie_created_at: remember_device_cookie.created_at)
-    redirect_to after_otp_verification_confirmation_url unless reauthn?
+    redirect_to after_sign_in_path_for(current_user) unless reauthn?
```

Comment:
I noticed that since it's defined on `application_controller`, the `after_sign_in_path_for` method would already have access to `current_user` and not need it passed as an arg, but then looking at it's source, the `_user` is unused!

https://github.com/18F/identity-idp/blob/b121550fa7a3d9096ddbed34a8c13f77fe3649b0/app/controllers/application_controller.rb#L232

Looks like the [method comes from devise](https://github.com/heartcombo/devise/blob/4b72064bfcf076478c5c87818b9536b203f6584f/lib/devise/controllers/helpers.rb#L206) but what if we redefined it to have a default value for user and then just stopped passing `current_user` everywhere to simplify?


```suggestion
    redirect_to after_sign_in_path_for unless reauthn?
```

```ruby
def after_sign_in_path_for(_scope = nil)
```

Reply:
I think it would be good to try this, but probably better tackled in a separate PR?